### PR TITLE
chore: upgrade dependencies

### PR DESCRIPTION
**Pull request by murali-shris**
**- What I did**
- upgraded dependencies for at_commons major version
**- How I did it**
- updated pubspec and changelog
**- How to verify it**
- tests should pass

### CODE REVIEW

**Comment by murali-shris** [comment]
@XavierChanth once this PR is merged, I need your help in publishing noports.
is this the correct link for noports dart package?
https://pub.dev/packages/noports_core/versions
no ports core in changelog is 6.1.x..But i see only 6.0.2 published

**Comment by gkc** [comment]
@murali-shris  We will hold off on publishing noports_core until we do the next full release of NoPorts product

**Review by gkc — APPROVED** [review]

**Pull request merged**
